### ISSUE TITLE
Int metric with SumAggregation fails to export to SD

### ISSUE THREAD

**Issue by colincadams** [census-instrumentation/opencensus-python#437]
The metric is [created as `MetricDescriptorType.CUMULATIVE_INT64`](https://github.com/census-instrumentation/opencensus-python/blob/d0409fceed79c206fdb27847aeb1176ce372cefe/opencensus/stats/metric_utils.py#L53), but a [float value is set on the point](https://github.com/census-instrumentation/opencensus-python/blob/d0409fceed79c206fdb27847aeb1176ce372cefe/opencensus/stats/exporters/stackdriver_exporter.py#L234).

It looks like there is some implementation missing in the stackdriver exporter. It seems like we keep breaking out the conditional based on the types of the aggregations, as was done for `DistributionAggregationData`. This will tell us what property we need to access on the aggregation object to get the result. 

(Right now it looks to me like it decides the value to send, for aggregations other than distributions, by looking at the value of the first tag column? If it can be converted to a float then sends that tag value as a float, otherwise it sends it as a string. Was this just a temporary solution early on? Given that the test only uses distribution, this seems like the most likely answer to me?)

The odd thing is that for sums `SumAggregationDataFloat` is always used, whether or not it is an `int` or a `float`. It does look like the same implementation can be used for both, so the `Float` suffix should likely just be stripped.

Then we can decide which field to set on value either by the type of the aggregated data, or by looking at `v_data.view.measure`, as is done in `create_metric_descriptor`.

*I'm happy to take this work on, as long as my reasoning makes sense and I'm not missing something big!*

The error returned is this:
```
Traceback (most recent call last):
  File "/env/lib/python3.7/site-packages/opencensus/common/transports/async_.py", line 105, in _thread_main
    self.exporter.emit(data)
  File "/env/lib/python3.7/site-packages/opencensus/stats/exporters/stackdriver_exporter.py", line 145, in emit
    self.handle_upload(view_data)
  File "/env/lib/python3.7/site-packages/opencensus/stats/exporters/stackdriver_exporter.py", line 157, in handle_upload
    self.upload_stats(view_data)
  File "/env/lib/python3.7/site-packages/opencensus/stats/exporters/stackdriver_exporter.py", line 166, in upload_stats
    request[CONS_TIME_SERIES])
  File "/env/lib/python3.7/site-packages/google/cloud/monitoring_v3/gapic/metric_service_client.py", line 897, in create_time_series
    request, retry=retry, timeout=timeout, metadata=metadata
  File "/env/lib/python3.7/site-packages/google/api_core/gapic_v1/method.py", line 143, in __call__
    return wrapped_func(*args, **kwargs)
  File "/env/lib/python3.7/site-packages/google/api_core/retry.py", line 270, in retry_wrapped_func
    on_error=on_error,
  File "/env/lib/python3.7/site-packages/google/api_core/retry.py", line 179, in retry_target
    return target()
  File "/env/lib/python3.7/site-packages/google/api_core/timeout.py", line 214, in func_with_timeout
    return func(*args, **kwargs)
  File "/env/lib/python3.7/site-packages/google/api_core/grpc_helpers.py", line 59, in error_remapped_callable
    six.raise_from(exceptions.from_grpc_error(exc), exc)
  File "<string>", line 3, in raise_from
google.api_core.exceptions.InvalidArgument: 400 Request was missing field timeSeries[0].points[0].int64Value.
```

based on this:
```
m = measure.MeasureInt("m/m", "example", "1")
v = view.View("v/v", "example", [TagKey.REGION], m, aggregation.SumAggregation())
monitoring.register_views([m])
```

**Comment by songy23** [comment]
Thanks for catching the bug. You're right that `int` aggregated data should be set as Long value when exporting. (The full mapping table is https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/Export.md#aggregation-to-metric.) We'll fix this bug as part of the requirement when updating the Stackdriver exporter to consume metrics instead of view_data.

**Comment by c24t** [comment]
Fixed by #374, #438.
